lib/reporters/base.js

```diff
@@ -317,10 +322,14 @@ function Base (runner) {
  * the bundled reporters.
  *
  * @api public
+ *
+ * @param {Object} options
+ * @param {[boolean]} options.listFailures - if false, will not output failure information
  */
-Base.prototype.epilogue = function () {
+Base.prototype.epilogue = function (options) {
   var stats = this.stats;
   var fmt;
+  var listFailures = !options || options.listFailures !== false;
```

Comment:
if we're going with an object parameter, please do this instead:

```js
options = options || {};
var listFailures = options.listFailures !== false;
```

this will simplify things if we need to add options later.

Reply:
I'm thinking we might want to rename `listFailures` to `showFailures` to keep our verbs in sync with the reporter option...